apps/clearance_ui/src/components/PurlDetails.tsx

```diff
@@ -0,0 +1,56 @@
+// SPDX-FileCopyrightText: 2023 Double Open Oy
+//
+// SPDX-License-Identifier: MIT
+
+import React from "react";
+import {
+    Accordion,
+    AccordionContent,
+    AccordionItem,
+    AccordionTrigger,
+} from "@/components/ui/accordion";
+import { Badge } from "@/components/ui/badge";
+import { parsePurlAndQualifiers } from "@/helpers/parsePurlAndQualifiers";
+
+type Props = {
+    purl: string;
+};
+
+const PurlDetails = ({ purl }: Props) => {
+    const parsedPurl = parsePurlAndQualifiers(purl);
+    let packageName = parsedPurl.name + "@" + parsedPurl.version;
```

Comment:
I would prefer it if the namespace was also included here, I feel that it gives quite a lot of additional information about what package you are handling, and shouldn't be hidden, but of course, feel free to disagree. (For example there are a bunch of packages named "core" in different namespaces)

Reply:
I am trying to convey all absolutely necessary information in the package name, while conserving space for the Badge.  Have to try out some versions. Maybe include namespace, but put the path in the opening Accordion?